packages/components/src/renderers/AliasRenderer.tsx

```diff
@@ -37,13 +39,37 @@ export class AliasRenderer extends React.Component<AliasRendererProps, AliasRend
         };
     }
 
-    handleClick() {
+    static getEditableRawValue = (values: List<ValueDescriptor>): string[] => {
+        return values.reduce((arr, vd) => {
+            if (vd.display !== undefined && vd.display !== null) {
+                arr.push(vd.display);
+            }
+            return arr;
+        }, []);
+    };
+
+    static getEditableValue = (values: List<ValueDescriptor>): string => {
+        return values?.size === 0
+            ? ''
+            : values.first().display !== undefined
```

Comment:
Could display possibly be null here?

Reply:
I simplified this and it should handle undefined or null now